docs/3-virtual-machines-containers/3.2-local-development.md

```diff
@@ -48,10 +48,12 @@ Read [Creating a Base Box](https://www.vagrantup.com/docs/boxes/base) and [Creat
     ?> Many of Vagrant requirements are already taken care of for you in the example. It is up to you to figure out which one(s) you need to add.
 
 2. Add a [Vagrant `post-processor`](https://www.packer.io/docs/post-processors/vagrant/vagrant) to your Packer build template. Configure it to output both a VMWare Fusion image and a Vagrant box.
+- Add a vagrant post processor to automatically upload your box to your cloud acount
```

Comment:
```suggestion
- Add a vagrant post processor to automatically upload your box to your cloud acount
```

Seems odd that this comes before step 3 (create a VagrantCloud account) can we move this towards the end of this exercise and maybe mark it Extra Credit (see [this](https://devops-bootcamp.liatr.io/#/3-virtual-machines-containers/3.3-managing-infrastructure?id=extra-credit) for an example)

Reply:
I have moved this to extra credit as suggested.